tests/es/__init__.py

```diff
@@ -19,3 +19,14 @@ def elasticsearch_delete_index(index_name: str):
 def close_elasticsearch_connections(es_client):
     for conn in es_client.transport.connection_pool.connections:
         conn.pool.close()
+
+
+def clear_test_indexes(indexes: list, documents: list):
```

Comment:
Nothing in this method ensures that the indexes deleted are test indexes, so it is more appropriately named `clear_indexes`.

Reply:
changed